### ISSUE TITLE
Primo New UI

### ISSUE THREAD

**Issue by banetbi** [mehmetc/jQuery.PRIMO#14]
Will this library work with the Primo New UI? If not, will you be releasing a library compatible with the New UI?

Thanks

**Comment by mehmetc** [comment]
No, it wont work. But have a look at [primo-explore-dom](https://github.com/mehmetc/primo-explore-dom) it more or less does the same thing. 
- access to all components and it controllers, etc.
- user data with loans and fines
- records
- facets

And the [primo-explore-dom-ui](https://github.com/mehmetc/primo-explore-dom-ui) is build using this library. This library gives you a graphical browser of all rendered components with the possibility to look inside them and see the services used.

I'm at __ELUNA2018__ if you want to meet?

Mehmet
